### ISSUE TITLE
Arme runique densante + frappe runique

### ISSUE THREAD

**Issue by Elahpascriptelaspeblood** [way-of-elendil/3.3.5#1061]
**Description**
La DRW ne fait aucune frappe runique lorsque le dk en fait une.

J'en profite aussi pour dire que la DRW ne fait aucun coup critique alors que je joue avec plus de 40% de coup critique et que elle est sensé se basé sur mes stats apparemment donc voilà. 
Elle fait aussi bcp de parry et dodge alors que j'ai le cap expertise en pve et en pvp, elle fait aussi des ratés alors que j'ai mon cap touché en pve et en pvp voilà merci, en espérant que se soit fixé !

Je rappel que c'est sensé être un atout du dk sang et non une enclume accroché à la jambe c'est tout de même un pt de talent TOUT AU FOND de l'arbe sang donc doit y avoir un minimum de dommage déjà que les frappes au coeur font du 1k4 et les frappe de mort du 2k5 en pve, du 700 sur le HS et du 900 sur le DS en pvp, c'est un peu triste d'avoir un "burst" qui ne burst que dal 

**Étapes de reproduction**
- Classe : Dk
- Race : Orque
- Spécialisation : Sang
- Sort / Talent / Etc : DRW
- Circonstances (Donjon, Raid, PvP, autre)
- Equipement : Légendaire DEUILLELOMBRE 284 pour la tranche de dmg
- Buffs / Debuffs / Enchaînement de sort / Etc

**Comportement attendu**
La drw doit faire les frappes runique du dk.

**Captures d’écran / Vidéos**
![image](https://user-images.githubusercontent.com/72766795/97217225-39c3dc00-17c7-11eb-98c7-f9cb836b7620.png)
![image](https://user-images.githubusercontent.com/72766795/97217284-53fdba00-17c7-11eb-92ff-57d5d20dbe38.png)

**Ressources additionnelles**
Fournissez des liens vers forums / woehead / wowhead / vidéos / guides / etc qui pourraient aider a confirmer qu'il s'agit d'un bug et a corriger le problème. Notez bien que l'officiel étant maintenant sur une version supérieure a 3.3.5, il est utile de vérifier la date de l'information, afin d'assurer qu'elle correspond a notre version. Notez que bien qu'un lien vers un serveur prive est utile, il ne peut pas être utilise comme preuve de la présence / absence d'un bug.

**Comment by Elahpascriptelaspeblood** [comment]
Je souhaiterai avoir une réponse svp 👍

**Comment by Ischjgt** [comment]
Patch 3.0.8 (2009-01-20): The weapon no longer echoes the Raise Dead and Army of the Dead spells. In addition it will now more closely match the master's damage and will cast Rune Strike when its master does. Costs 40 Runic Power - (Old 50) 

Par elle fait bien des critiques quand le DK fait des crit

**Comment by Elahpascriptelaspeblood** [comment]
bah c'est pas ce que m'a dit josh, regarde l'autre BT que j'ai fait ya quelque semaine :o

**Comment by Elahpascriptelaspeblood** [comment]
genre c'est une copie de mes stats avec 50% de dmg en moins appliqué

**Comment by Elahpascriptelaspeblood** [comment]
après si quand je crit elle crit moi ca me vas :)

**Comment by Justforsoloq** [comment]
Non il a pas vraiment dit ça, il a juste expliqué comment la DRW était calculé.
Et c'est un modif 50% sur les damages globaux (pas 50% de dmg en moins encore une fois). Et oui la drw peut crit, si Elah tu avais posté les logs que je t'avais donné dans ton BT ça aurait suffit à prouver que la drw est censé pouvoir crit.

**Comment by Elahpascriptelaspeblood** [comment]
bah ca était posté sur l'autre BT

**Comment by Elahpascriptelaspeblood** [comment]
apres si il faut remettre le liens je peux le faire ^^

**Comment by Justforsoloq** [comment]
Et la DRW ne crit pas quand tu crit. Elle a juste ton taux de crit snapshot quand tu l'invoques et donc a ton % de crit quand elle tape. Elle crit sur sa plage à elle, pas quand toi tu crit.

**Comment by Elahpascriptelaspeblood** [comment]
oui je sais c'est bien ce que m'avais dit josh, mais là la drw ne crit pas du tout sur woe c'est pour ca que je fait la remarque

**Comment by Justforsoloq** [comment]
Yep pour ça que je te dis de poster les liens que je t'ai filé qui prouve sur ce post que la drw crit.

**Comment by Elahpascriptelaspeblood** [comment]
http://www.worldoflogs.com/reports/0abwf6ft1azn5ney/details/10/?s=5906&e=6158 (BDK)
http://www.worldoflogs.com/reports/0abwf6ft1azn5ney/details/73/?s=5906&e=6158 (DRW du BDK)

**Comment by Elahpascriptelaspeblood** [comment]
c'est good merci à toi pour les avoir link ;)

**Comment by Elahpascriptelaspeblood** [comment]
Des nouvelles concernant l'avancer du BT ?

**Comment by Elahpascriptelaspeblood** [comment]
Des nouvelles concernant le BT ? 
Je sais bien qu'un des sorts de burst majeur du BDK n'est pas dans vos priorités mais une réponse serai apprécié.
L'édit date de 2020, un tout petit peu de respect et d'attention sur l'habilité serai grandement bienvenu.

**Comment by Justforsoloq** [comment]
Si jamais petit édit pour dire qu'il y a aussi un up sur l'arme runique des bdk en même temps que le nerf 
(% de crit quasiment inexistant de la drw, mais une vitesse d'attaque de la drw bien + grande que ce qu'elle devrait avoir)

**Comment by Elahpascriptelaspeblood** [comment]
C'est bien ce qui est souhaité ici Melendder, malheureusement ça ne répond pas à ma question d'origine. Est-ce qu'il y a des nouvelles concernant l'avancé de ce BT ?
